internal/dag/policy.go

```diff
@@ -169,3 +170,22 @@ func max(a, b uint32) uint32 {
 	}
 	return b
 }
+
+func prefixReplacementsAreValid(replacements []projcontour.ReplacePrefix) error {
+	prefixes := map[string]struct{}{}
```

Comment:
why not make this 

 prefixes := map[string]bool 

Then the logs below becomes

if prefixes[r.Prefix] { ... }

Reply:
Fixed.